### PR TITLE
E2e mpe test e

### PR DESCRIPTION
**Pull request by prastoin**
close https://github.com/AdonisEnProvence/MusicRoom/issues/384

### CODE REVIEW

**Comment by prastoin** [comment]
The created is atm failing because we need to fix this [issue](https://github.com/AdonisEnProvence/MusicRoom/issues/375) before merging this PR

File: packages/client/e2e/mpe/test-e.spec.ts [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+    });
+
+    await exportMpeRoomToMtvRoom({
+        mtvRoomName: 'cocorico',
+        page,
+    });
+
+    const addedTrackTestID = `mtv-${addedTrack.id}-track-card-container`;
+    const addedTrackMtvCard = page.locator(
+        withinMpeRoomScreen(`css=[data-testid$="${addedTrackTestID}"]`),
```
Comment (Devessier) [inline packages/client/e2e/mpe/test-e.spec.ts:242]: I think we want to assert that the song can be found on the Music Player? Not in the MPE room?

**Review by Devessier — APPROVED** [review]
Nice job!
I left a comment we should discuss before merging.

**Review by Devessier — APPROVED** [review]
Nice job!
We can merge this last E2E test for MPE rooms!

**Pull request merged**